assets/images/Zoom - Blue.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Little pet-peevish but I think you should rename this file so the name doesn't contain any spaces (something I do my best to universally avoid). Maybe just `zoom.svg` would work.